src/data_distribution/first_touch_distribution.hpp

```diff
@@ -11,6 +11,9 @@
 
 #include "base_distribution.hpp"
 
+/** @brief The size used by each page in the owners_dir */
+const std::size_t page_info_size = 3;
```

Comment:
could we document here how the 3 is computed, or at least what kind of changes would also require this number to change?